python/mxnet/gluon/data/dataloader.py

```diff
@@ -74,6 +74,38 @@ def reduce_ndarray(data):
 
 ForkingPickler.register(nd.NDArray, reduce_ndarray)
 
+if sys.platform == 'darwin' or sys.platform == 'win32':
+    def rebuild_ndarray(*args):
```

Comment:
can you rename `reduce_ndarray` and `rebuild_ndarray` to `np_xxx`? Reusing the same function may cause some problems even after register

Reply:
Changed to `reduce_np_ndarray ` and `rebuild_np_ndarray `, does that sound good to you?